lib/dal/src/workspace_snapshot/node_weight/input_socket_node_weight/v1.rs

```diff
@@ -155,7 +155,7 @@ impl SiNodeWeight for InputSocketNodeWeightV1 {
     fn node_hash(&self) -> ContentHash {
         let mut content_hasher = ContentHash::hasher();
         content_hasher.update(self.arity.to_string().as_bytes());
-        content_hasher.update(self.content_address.to_string().as_bytes());
+        content_hasher.update(self.content_address.content_hash().to_string().as_bytes());
```

Comment:
Turns out that `self.content_address.to_string()` is just `SchemaVariant`, and doesn't actually include any of the hash. 🙃